engine/core/src/main/java/com/torodb/core/transaction/metainf/ImmutableMetaCollection.java

```diff
@@ -132,6 +132,7 @@ public ImmutableMetaCollection immutableCopy() {
             .getName()) != null)
         .flatMap(index -> Seq.seq(index.iteratorMetaDocPartIndexesIdentifiers(docPart))
             .filter(identifiers -> identifiers.contains(newField.getIdentifier()))
+            // Fix for Eclipse compiler, please do not remove explicit generics
```

Comment:
<joke>
you should add something like "because Eclipse compiler is very dumb" to the comment
</joke>